index.bs

```diff
@@ -1241,12 +1251,24 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     An [=authorization gesture=] is a [=ceremony=] component often employed to indicate [=user consent=].
 
 : <dfn>User Handle</dfn>
-:: The user handle is specified by a [=[RP]=], as the value of
-    <code>{{PublicKeyCredentialCreationOptions/user}}.{{PublicKeyCredentialUserEntity/id}}</code>, and used to [=credentials map|map=] a specific [=public key credential=]
-    to a specific user account with the [=[RP]=]. Authenticators in turn [=credentials map|map=] [=RP ID=]s and user handle pairs
-    to [=public key credential sources=].
+:: A user handle is an identifier for a user, specified by the [=[RP]=] as
+    <code>{{PublicKeyCredentialCreationOptions/user}}.{{PublicKeyCredentialUserEntity/id}}</code>
+    during [=registration=].
+    [=Discoverable credentials=] store this identifier and return it as
+    <code>{{PublicKeyCredential/response}}.{{AuthenticatorAssertionResponse/userHandle}}</code>
+    in [=authentication ceremonies=] started with an [=list/empty=]
+    <code>{{PublicKeyCredentialRequestOptions/allowCredentials}}</code> argument.
+    Its main use is to identify the user in such [=authentication ceremonies=],
+    but the [=credential ID=] could be used instead.
+    The main differences are
+    that the [=user handle=] is chosen by the [=[RP]=] and MAY be the same for all of a user's [=credentials=],
```

Comment:
This suggestion is per my overall comment of trying to be specific about the notion of mapping creds to user accounts via user handle
```suggestion
    that the [=user handle=] is chosen by the [=[RP]=], such that there is a 1:1 mapping between user handles and user accounts at the RP.  Note that  if the user registers multiple credentials with the RP using the same account, all of those credentials ought to have the same user handle value.
```
...or something along those lines.

Reply:
Thanks, I think I managed to condense this down a bit while still capturing the same ideas.